packages/bierzo-wallet/src/routes/balance/components/index.tsx

```diff
@@ -72,7 +72,7 @@ const BalanceLayout = ({ name, tokens, onSendPayment, onReceivePayment }: Props)
       <Block bgcolor={theme.palette.background.paper} height="unset" width={450}>
         <Block padding={4} display="flex" flexDirection="column">
           <Typography variant="h5" align="center" weight="light">
-            {name ? name : '--'}
+            {iovAddress ? iovAddress : 'Get your human readable address.'}
```

Comment:
Can you check with @kmw101 and @webmaster128 if they give thumbs up for the 'Get your human-readable address' text?

Reply:
👌